packages/api/src/modules/etablissements/etablissements.service.ts

```diff
@@ -89,6 +89,10 @@ export class EtablissementsService {
         return await documentsService.getDocumentBySiret(siret);
     }
 
+    async getRibs(siret: Siret) {
+        return await documentsService.getRibsBySiret(siret);
+    }
```

Comment:
```suggestion
    getRibs(siret: Siret) {
        return documentsService.getRibsBySiret(siret);
    }
```
la même chose en moins verbeux mais ça change rien